docs/proposals/http-tunnel-expose-strategy.md

```diff
@@ -0,0 +1,149 @@
+# **Exposing control-plane services using SNI and HTTP tunneling**
+
+Author: Youssef Azrak (@youssefazrak), Iacopo Rozzo (@irozzo-1A)
+
+Status: Draft proposal.
+
+
+## **Goals**
```

Comment:
No need to have this text be a heading AND bold.

Reply:
Fixed